src/reconcile/rec_write.c

```diff
@@ -598,8 +600,7 @@ __rec_init(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags, WT_SALVAGE_COO
      * Add a write barrier to make it more likely that a thread adding an update will see this state
      * change.
      */
-    page->modify->page_state = WT_PAGE_DIRTY_FIRST;
-    WT_FULL_BARRIER();
```

Comment:
I think a write barrier here is enough. We should not use a full barrier to slow down the execution.

Reply:
I don't understand this comment. That is the existing code, and I made no recommendation to change it.

Reply:
I'm referring your comment on the jira ticket. You don't need to change this.

Reply:
Thanks,. Perhaps you might consider commenting in the issue where there is context.